index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="conteiner">
+        <div class="main-left">
+            <div class="main-left-content">
+                <h1>Join the Stack Overflow community</h1>
+                <ul>
+                    <li class="text-medium">Get unstuck — ask a question</li>
+                    <li class="text-medium">Unlock new privileges like voting and commenting</li>
+                    <li class="text-medium">Save your favorite tags, filters, and jobs</li>
+                    <li class="text-medium">Earn reputation and badges</li>
+                </ul>
+                <p class="text-small">Collaborate and share knowledge with a private group for FREE.</p>
+                <a class="left-a" target="_blank" href="https://stackoverflow.co/teams/?utm_source=so-owned&utm_medium=product&utm_campaign=free-50&utm_content=public-sign-up" class="text-small">Get Stack Overflow for Teams free for up to 50 users.</a>      
+            </div>
+        </div>
+        <div class="main-right">
+            <div class="Sign-Up-Buttons">
```

Comment:
It's better to use the same case for all classes, e.g. use the 'kebab-case' as you did in all other places ('sign-up-buttons').